app/product_notices.go

```diff
@@ -4,20 +4,21 @@
 package app
 
 import (
-	"github.com/mattermost/mattermost-server/v5/store"
 	"net/http"
 	"reflect"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/mattermost/mattermost-server/v5/store"
```

Comment:
Let's fix it properly while we are here. We should split them into 3 groups - stdlib, mm packages, 3rd party packages.

Reply:
Fixed